src/Identities.ts

```diff
@@ -33,6 +45,11 @@ export class Identities {
    */
   public registerIdentity: ProcedureMethod<RegisterIdentityParams, Identity>;
 
+  /**
+   * Create a new Portfolio under the ownership of the Current Identity
```

Comment:
(nit)
```suggestion
   * Create a new Portfolio under the ownership of the signing Identity
```
I think this makes it a bit more clear compared to "Current".